apps/meteor/tests/end-to-end/api/24-methods.js

```diff
@@ -123,6 +126,276 @@ describe('Meteor.methods', function () {
+					});
+			});
+		});
+
+		(IS_EE ? describe : describe.skip)('[@getReadReceipts] EE', () => {
+			let user = null;
+			let userCredentials = null;
+			let room = null;
+			let firstMessage = null;
+			let firstThreadMessage = null;
+
+			const roomName = `methods-test-channel-${Date.now()}`;
+			before(async () => {
+				await updateSetting('Message_Read_Receipt_Enabled', true);
+				await updateSetting('Message_Read_Receipt_Store_Users', true);
+
+				user = await createUser();
+				userCredentials = await login(user.username, password);
+				room = (await createRoom({ type: 'p', name: roomName, members: [user.username] })).body.group;
+				firstMessage = (await sendSimpleMessage({ roomId: room._id })).body.message;
+				firstThreadMessage = (await sendSimpleMessage({ roomId: room._id, tmid: firstMessage._id })).body.message;
+			});
+
+			after(async () => {
+				await deleteRoom({ type: 'p', roomId: room._id });
+				await deleteUser(user);
+			});
+
+			it("should return the sender's read receipt for a message sent in the main room", async () => {
+				await request
+					.post(methodCall('getReadReceipts'))
+					.set(credentials)
+					.send({
+						message: JSON.stringify({
+							method: 'getReadReceipts',
+							params: [{ messageId: firstMessage._id }],
+							id: 'id',
+							msg: 'method',
+						}),
+					})
+					.expect('Content-Type', 'application/json')
+					.expect(200)
+					.expect((res) => {
+						expect(res.body).to.have.a.property('success', true);
+						expect(res.body).to.have.a.property('message').that.is.a('string');
+						console.log('res.body', res.body);
+
+						const data = JSON.parse(res.body.message);
+						console.log('data', data);
+						expect(data).to.have.a.property('result').that.is.an('array');
+						expect(data.result.length).to.equal(1);
+						expect(data.result[0]).to.have.property('userId', credentials['X-User-Id']);
+					});
+			});
+
+			it("should return the sender's read receipt for a message sent in a thread", async () => {
+				await request
+					.post(methodCall('getReadReceipts'))
+					.set(credentials)
+					.send({
+						message: JSON.stringify({
+							method: 'getReadReceipts',
+							params: [{ messageId: firstThreadMessage._id }],
+							id: 'id',
+							msg: 'method',
+						}),
+					})
+					.expect('Content-Type', 'application/json')
+					.expect(200)
+					.expect((res) => {
+						expect(res.body).to.have.a.property('success', true);
+						expect(res.body).to.have.a.property('message').that.is.a('string');
+
+						const data = JSON.parse(res.body.message);
+						expect(data).to.have.a.property('result').that.is.an('array');
+						expect(data.result.length).to.equal(1);
+						expect(data.result[0]).to.have.property('userId', credentials['X-User-Id']);
+					});
+			});
+
+			it("should read all main room's messages with the invited user", async () => {
```

Comment:
This isn't a test, but a requirement for the next few tests; 
Move them all to a new `describe` section that runs independently from this one, with it's own `before` sub-section.